docs/markers.rst

```diff
@@ -8,11 +8,16 @@ Mapcrafter allows you to add different markers easily to your rendered
 maps. Markers are organized in marker groups, this allows you show
```

Comment:
I like this new page!

Reply:
The best part was.... it was mostly what you'd already done!!